code/post.r

```diff
@@ -42,13 +42,13 @@ posterior.sim <- function(data, model, over = FALSE) {
   # by province
   for(ii in seq(p)) {
     # get the offset segment
-    toff <- rev(rev(data$off[ii, ])[seq(data$end[ii])])
+    toff <- data$off[ii, ][data$str[ii]:data$end[ii]]
```

Comment:
this pulls the series that starts with the first named, then goes to the 'end' of the series. but see actual values at in my next comment for the actual data file